conf/script/build-cortx-ha

```diff
@@ -20,7 +20,17 @@ set -eu -o pipefail
 
 HA_PATH="<HA_PATH>"
 
+log_file=${BUILD_CORTX_HA_LOG_FILE:-/var/log/build-cortx-ha.log}
```

Comment:
`log_file=${BUILD_CORTX_HA_LOG_FILE:-/var/log/build-cortx-ha.log}`

Here we are creating new file for each script which will create too many log file for same task.

can we use single generic file like `ha_resource_setup.log` single file for all script

Also add this to initial of file to identify script log
```
log "****************************************************************"
log "${0##*/}"
log "****************************************************************"
```

Reply:
The log file name is already indicating the script file for which this log is created. 
Will adding the same name in the log file help in any additional way ?

Reply:
Regarding 
>>can we use single generic file like ha_resource_setup.log single file for all script
we are already adding all the logs to the syslog by using the logger utility. So they are available in a single file there. 
I though that having separate files will help in locating logs from individual scripts.

However I am fine with having a single file if that is going to give better usability